deploy-to-local-docker.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+REMOTE_HOST="mrjc@docker-host.33.cleaver.org"
```

Shellcheck (0.10.0)

[warning] REMOTE_HOST appears unused. Verify use (or export if used externally).

(SC2034)